page-builder/hooks/html.md

```diff
@@ -26,6 +26,36 @@ echo apply_filters( 'siteorigin_panels_before_row', '', $panels_data['grids'][$g
 echo apply_filters( 'siteorigin_panels_after_row', '', $panels_data['grids'][$gi], $grid_attributes );
 ```
 
+
+### Inside Rows Before and After
+///
+
+You're able to output additional content inside of the rows and before/after the cells have been added. 
+
+```php
+// Row Container
+echo apply_filters( 'siteorigin_panels_inside_row_before', '', $row );
+// Cells
+echo apply_filters( 'siteorigin_panels_inside_row_after', '', $row );
+// Row Container End
+```
+
+### Inside Cells Before and After
+///
+
+Like with the Inside Before After Rows, you're able to output additional markup inside of the cells. This will give you chance to wrap the widgets added to the cell with addiitonal markup before/after the widget has rendered.
+
+```php
+// Row Container
+// Cell Container
+echo apply_filters( 'siteorigin_panels_inside_cell_before', '', $cell );
+// Widgets added to cell
```

Comment:
If we're using title case for the comments then perhaps this line could be:

"Widgets Added to Cell"